src/pages/ImportData.js

```diff
@@ -123,9 +124,10 @@ export const ImportDataOverview = ({ header, snapshots, isDataset, snapshotRespo
           ]),
         ])
       : url && div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
-    div({ style: { marginTop: '1rem' } }, [
-      !!url && isProtected(url, format)
-        ? h(Fragment, [
+    div(
+      { style: { marginTop: '1rem' } },
+      isProtectedData
+        ? [
             icon('warning-standard', { size: 15, style: { marginRight: '0.25rem' }, color: colors.warning() }),
             ' The data you chose to import to Terra are identified as protected and require additional security settings. Please select a workspace that has an Authorization Domain and/or protected data setting.',
```

Comment:
I think colloquially `or` can be used in cases where both things happen to be true (vs only in case where one or the other is true). So I would word this as 
```suggestion
            ' The data you chose to import to Terra are identified as protected and require additional security settings. Please select a workspace that has an Authorization Domain or protected data setting.',
```